site/src/sphinx/zookeeper.rst

```diff
@@ -0,0 +1,54 @@
+.. _`Apache ZooKeeper`: https://zookeeper.apache.org/
+.. _`com.linecorp.armeria.client.zookeeper`: apidocs/index.html?com/linecorp/armeria/client/zookeeper/package-summary.html
+.. _`com.linecorp.armeria.server.zookeeper`: apidocs/index.html?com/linecorp/armeria/server/zookeeper/package-summary.html
+
+Setting up a ZooKeeper EndpointGroup
+====================================
+You can put your Endpoints information into an zNode in `Apache ZooKeeper`_ Custer, as node value or as node's child value, like this:
+
+| as node value:
+|         /myProductionEndpoints -->192.168.1.10:8080:100,192.168.1.11:8080:100
```

Comment:
- Missing space after `-->`
- Could you add some explanation about the syntax? e.g. what `100` means and that the weight value can be omitted because it's optional.